lib/stitches/cache_injector.rb

```diff
@@ -0,0 +1,32 @@
+require 'lru_redux'
+
+module Stitches::CacheInjector
+  MAX_ITEMS_IN_CACHE = 100
+  CACHE_DURATION_IN_SECONDS = 5 * 60 # five minutes
+
+  def self.inject
+    ::ApiClient.define_singleton_method 'api_key_cache' do
+      @api_key_cache ||= LruRedux::TTL::Cache.new(
```

Comment:
❓ Since consuming apps are restarted whenever a key is added or removed, do we need to use a TTL cache?

Reply:
I don't think the provider application is restarted when a key is added or removed.

Reply:
Yeah, of course. Thanks 😂